pkg/controller/controller.go

```diff
@@ -450,6 +466,11 @@ func (ctrl *resizeController) resizeVolume(
 	if err != nil {
 		return newSize, fsResizeRequired, err
 	}
+	if fsResizeRequired {
```

Comment:
This will update same PV twice. Can we do that in same function `updatePVCapacity` and may be supply it with additional parameters that are necessary to add the annotation.

Reply:
That sounds good. Will do.